### PR TITLE
CICD Samples refresh with Bash scripts

### PR DESCRIPTION
**Pull request by jesp209i**
## Description

- Added how to set up CICD using bash scripts
- gave the Samples README a refresh to reflect updated guides
- moved the old guides 

## Type of suggestion

* [ ] Typo/grammar fix
* [x] Updated outdated content
* [x] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

Umbraco Cloud, CI/CD

### CODE REVIEW

**Comment by jesp209i** [comment]
All Linting errors are coming from the existing documentation, not the added one :)
azure-devops.md was renamed to azure-devops-old.md
Parts of README.md was copied into old-setup.md

As an alternative we could remove all the "old" parts?

File: umbraco-cloud/set-up/project-settings/umbraco-cicd/samplecicdpipeline/github-actions.md [hunk 1]
```diff
@@ -1,71 +1,238 @@
...
+Download the provided sample scripts as ZIP from the [GitHub repository](https://github.com/umbraco/Umbraco.Cloud.CICDFlow.Samples/tree/main). Click on "Code" and then choose "Download ZIP". Then unzip it and use those files for the next steps.
+
+{% endhint %}
+
+### Optional: Run a deployment to confirm
+
+With everything set up, you may want to confirm that Umbraco Cloud reflects the changes you are sending via your pipeline.
+
+While working on you project locally, add a new Document type.
+- Commit the change to `main` and push to your repository.
```
Comment (alina-tincas) [inline umbraco-cloud/set-up/project-settings/umbraco-cicd/samplecicdpipeline/github-actions.md:192]: shouldn´t this be changed as "- Commit the change to `main` (or `master` if you did not change the branch name) in Branch and push to your repository."?

File: umbraco-cloud/set-up/project-settings/umbraco-cicd/samplecicdpipeline/azure-devops.md [hunk 2]
```diff
@@ -1,178 +1,222 @@
...
 
-This stage is responsible for creating a new deployment and ensuring that it can be initiated. For more details, refer to the "Create Deployment API" section.
+### Optional: Run a deployment to confirm
 
-The deployment artifact, a zip file containing the source files, is uploaded at this stage. For more information, see the "Upload Zip Source File" section.
+With everything set up, you may want to confirm that Umbraco Cloud reflects the changes you are sending via your pipeline.
 
-The deployment is then initiated. For further details, consult the "Start Deployment" section.
+While working on your project locally, add a new Document type.
+- Commit the change to `main` and push to your repository.
```
Comment (alina-tincas) [inline umbraco-cloud/set-up/project-settings/umbraco-cicd/samplecicdpipeline/azure-devops.md:176]: shouldn´t this be"- Commit the change to `main` (or `master` if you did not change the branch name) in Branch and push to your repository."

File: umbraco-cloud/set-up/project-settings/umbraco-cicd/samplecicdpipeline/github-actions.md (hunk 1 above)
Comment (alina-tincas) [reply umbraco-cloud/set-up/project-settings/umbraco-cicd/samplecicdpipeline/github-actions.md:192]: also, unlike azure devops, if the branch was not changes to main and is still master, there were no other extra steps like changing some yaml files to update branch name right?

**Review by alina-tincas — CHANGES_REQUESTED** [review]
Heya @jesp209i happy new year 🎉 

Thank you for the PR #h5YR

I have just few suggestions, can you please take a look a them and let me know what you think about them? 😊

File: umbraco-cloud/set-up/project-settings/umbraco-cicd/samplecicdpipeline/github-actions.md (hunk 1 above)
Comment (jesp209i) [reply umbraco-cloud/set-up/project-settings/umbraco-cicd/samplecicdpipeline/github-actions.md:192]: GitHub also has the section about changing master branch to mail - optionally

**Review by alina-tincas — APPROVED** [review]

**Comment by alina-tincas** [comment]
Thank you for making the changes! All looks good so am merging it in 😊 🎉

**Pull request merged**